definitions/npm/ramda_v0.26.x/flow_v0.76.x-/test_ramda_v0.26.x_list.js

```diff
@@ -211,9 +211,29 @@ const str: string = "hello world";
   const groupedWith: Array<Array<number>> = _.groupWith(x => x > 1, ns);
   const groupedWith1: Array<Array<string>> = _.groupWith(x => x === "one")(ss);
 
-  const xOfXs: ?number = _.head(ns);
-  const xOfXs2: ?number = _.head(ns);
-  const xOfStr: string = _.head(str);
+  describe('head', () => {
+    describe('with array', () => {
+      it('should returns MaybeOf element type', () => {
+        const fn = (arr: number[]) => {
+          const xOfXs: ?number = _.head(arr);
+          //$ExpectError
+          const xOfXs: number = _.head(arr);
```

Comment:
This one surprises me actually, because array accessors are [considered unsafe](https://flow.org/en/docs/types/arrays/#toc-array-access-is-unsafe), I would've expected it to return a non-maybe type.

Reply:
Nothing to do here, I just didn't expect `$ElementType` to behave in this way.